consensus/obcpbft/config.yaml

```diff
@@ -49,6 +49,10 @@ general:
 
         # How long may a view change take
         viewchange: 2s
+
+        # Interval to send "keep-alive" null requests.  Set to 0 to disable.
+        nullrequest: 0s
```

Comment:
We can change it later if we want to change the default.  We probably will also want to change batch size and consensus default (noop).